src/Web/Grand.Web.Common/Extensions/DateTimeMappingExtensions.cs

```diff
@@ -1,12 +1,16 @@
 ﻿using Grand.Business.Core.Interfaces.Common.Directory;
 
-namespace Grand.Web.Admin.Extensions.Mapping
+namespace Grand.Web.Common.Extensions
 {
     public static class DateTimeMappingExtensions
     {
         public static DateTime? ConvertToUserTime(this DateTime? datetime, IDateTimeService dateTimeService)
         {
-            if (datetime is { Kind: DateTimeKind.Utc }) datetime = dateTimeService.ConvertToUserTime(datetime.Value, TimeZoneInfo.Utc, dateTimeService.CurrentTimeZone);
```

Comment:
Please return back this notation:
if (datetime is { Kind: DateTimeKind.Utc })

Reply:
Done.

Reply:
Can you please check now.